knowledge-base/how-to-switch-reports-in-kendo-mvc-modal-window.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Switch Reports Displayed in Kendo MVC Modal Window
+description: How to switch reports effectively in an Html5 MVC Report Viewer hosted in a Kendo MVC modal window
```

Comment:
Grammar: I would leave articles a/an
How to switch reports effectively in Html5 MVC Report Viewer hosted in Kendo MVC modal window